package.json

```diff
@@ -168,6 +168,8 @@
     "underscore": "^1.13.1"
   },
   "devDependencies": {
+    "@dword-design/eslint-plugin-import-alias": "^4.0.8",
```

Comment:
This is rather small npm [package](https://www.npmjs.com/package/@dword-design/eslint-plugin-import-alias) with 4k downloads in the last week. Are there any other alternatives?

Reply:
reason: https://github.com/Expensify/App/pull/26605#issuecomment-1712899914

Reply:
There is one but, it's not actively maintained.